tests/python/quantization/test_quantization.py

```diff
@@ -414,6 +414,57 @@ def check_quantized_flatten(shape, qdtype):
         check_quantized_flatten((10, 15, 18), qdtype)
         check_quantized_flatten((3, 4, 23, 23), qdtype)
 
+@with_seed()
+def test_quantized_act():
```

Comment:
Do we have cases for non-relu activation type?

Reply:
Negative test cases are not necessary there, it well prompt that _contrib_quantized_act only supports act_type=relu for now.

Reply:
Fine. I'm personally okay about there is no non-relu cases in UT. But do you mind running one on your machine and paste the error log here? I just want to make sure that the error happens at a appropriate place and the message is accurate.

Reply:
Sure, the error log just like below when change activation type from relu to no-relu type (e.g. sigmoid):
[16:03:13] src/operator/quantization/quantized_activation.cc:54: _contrib_quantized_act only supports act_type=relu for now
Aborted